internal/layer2/announcer.go

```diff
@@ -63,10 +65,21 @@ func (a *Announce) updateInterfaces() {
 	a.Lock()
 	defer a.Unlock()
 
+	var excludeRegexp *regexp.Regexp
```

Comment:
We should initialize this only once and use it instead of creating it every time we need to use it.